recipes/incbin/all/conanfile.py

```diff
@@ -22,20 +21,40 @@ class IncbinConan(ConanFile):
     def layout(self):
         basic_layout(self, src_folder="src")
 
-    def package_id(self):
-        self.info.clear()
+    def configure(self):
+        self._is_msvc = is_msvc(self)
 
-    def validate(self):
-        if is_msvc(self):
-            raise ConanInvalidConfiguration("Currently incbin recipe is not supported for Visual Studio because it requires external command 'incbin'.")
+    def package_id(self):
+        if self._is_msvc:
```

Comment:
```suggestion
        if is_msvc(self.info):
```
Can drop the `self._is_msvc = is_msvc(self)` as well then.

Reply:
i need self._is_msvc: in source() method because
        
        ConanException: 'self.info' access in 'source()' method is forbidden
        
        neither
        
        ConanException: 'self.settings' access in 'source()' method is forbidden